plugins/scaffolder-node/src/tasks/types.ts

```diff
@@ -181,7 +181,20 @@ export interface TaskBroker {
   get(taskId: string): Promise<SerializedTask>;
 
   list?(options?: {
+    /**
+     * @deprecated Please use the `filters` parameter instead
```

Comment:
Realized that this does not show up in the API report and editor as strikethrough unfortunately.

The thing we can do is to overload list and deprecate the old list. The new list method need to be added before the deprecated one to show up.

```ts
  list?(options?: {
    filters?: {
      createdBy?: string | string[];
      status?: TaskStatus | TaskStatus[];
    };
    limit?: number;
    offset?: number;
    order?: { order: 'asc' | 'desc'; field: string }[];
  }): Promise<{ tasks: SerializedTask[]; totalTasks?: number }>;

  /**
   * @deprecated Make sure to pass `createdBy` and `status` as filters to the `list` method instead
   */
  list?(options?: {
    createdBy?: string;
    status?: TaskStatus;
    filters?: {
      createdBy?: string | string[];
      status?: TaskStatus | TaskStatus[];
    };
    limit?: number;
    offset?: number;
    order?: { order: 'asc' | 'desc'; field: string }[];
  }): Promise<{ tasks: SerializedTask[]; totalTasks?: number }>;
```